### PR TITLE
[Onnx Operators] Celu

### PR DESCRIPTION
**Pull request by CircleSpin**
The celu onnx operator has been implemented. https://github.com/onnx/onnx/blob/master/docs/Operators.md#celu

@mbrookhart @jwfromm

### CODE REVIEW

File: tests/python/frontend/onnx/test_forward.py [hunk 1]
```diff
@@ -4668,7 +4668,7 @@ def verify_eyelike(indata):
     "test_cast_FLOAT_to_BFLOAT16",
     "test_cast_FLOAT_to_STRING",
     "test_cast_STRING_to_FLOAT",
-    "test_celu",
+    # "test_celu",
```
Comment (mbrookhart) [inline tests/python/frontend/onnx/test_forward.py:4671]: Just remove, don't comment.

File: python/tvm/relay/frontend/onnx.py [hunk 2]
```diff
@@ -23,6 +23,7 @@
 import numpy as np
 import tvm
 from tvm.ir import IRModule
+from tvm.relay.transform.transform import ConvertLayout
```
Comment (mbrookhart) [inline python/tvm/relay/frontend/onnx.py:26]: Unused, remove

File: python/tvm/relay/frontend/onnx.py [hunk 3]
```diff
@@ -3607,6 +3608,7 @@ def _get_convert_map(opset):
         "ConvInteger": ConvInteger.get_converter(opset),
         # Random number generation.
         "RandomUniform": RandomUniform.get_converter(opset),
+        "Celu": Celu.get_converter(opset),
```
Comment (mbrookhart) [inline python/tvm/relay/frontend/onnx.py:3611]: Maybe stick this higher in the list by Relu to keep similar ops together?

**Review by mbrookhart — APPROVED** [review]
Nitpicks, LGTM

**Comment by junrushao1994** [comment]
The PR looks good to me :-)

The CI is unblocked by this commit https://github.com/apache/tvm/commit/f5661f4e78cae5f62bc172559040576cd2fd20f2, so I just rebased this PR to main:HEAD to retrigger the CI so that we could get it merged ASAP

**Comment by junrushao1994** [comment]
Thanks @CircleSpin @mbrookhart! It’s merged!

**Comment by CircleSpin** [comment]
Thank you Junru for the rebase :)

**Pull request merged**